mslib/msidp/idp_conf.py

```diff
@@ -36,28 +35,21 @@
 from saml2.saml import NAME_FORMAT_URI
 from saml2.saml import NAMEID_FORMAT_PERSISTENT
 from saml2.saml import NAMEID_FORMAT_TRANSIENT
+from saml2.sigver import get_xmlsec_binary
+
+from mslib.mscolab.conf import mscolab_settings
```

Comment:
msidp is a separate module from mscolab. With msui we deliberately don't use anything from mscolab. Should msidp likewise not depend on mscolab internals?